### DISCUSSION TITLE
What is the minimal oauth scope necessary for /api/v1/apps/verify_credentials ?

### DISCUSSION THREAD

**Discussion by johnspurlock** [mastodon/mastodon#17121] [Q&A] [answered]
Hi there, I'm trying to create a client app that can only post comments, but also want to call `/api/v1/apps/verify_credentials` to verify the access token after initial oauth setup.

I'm assuming `write:statuses` will work for posting comments, but what is the appropriate granular oauth scope for `/api/v1/apps/verify_credentials`?  I would have expected `read:accounts` based on #11263 but that doesn't work on a `v3.4.4` server.

```
{
  "error": "This action is outside the authorized scopes"
}
```

**Comment by trwnh** [comment, marked as answer]
It appears to be `read` with no smaller scope, if I'm reading this correctly? https://github.com/mastodon/mastodon/blob/6c8c031bcd859510bf4f58c355dce7b6b4346401/app/controllers/api/v1/apps/credentials_controller.rb#L4

**Comment by johnspurlock** [comment]
Looking at the api some more, I think I should be using `/api/v1/accounts/verify_credentials` for my use case (the `/accounts/` version not the `/apps/` version), since I'm dealing with user-level access tokens.  

The `/api/v1/accounts/verify_credentials` endpoint _does_ work with `read:accounts`.  Thanks for looking into it!
